src/setuptools_pyproject_migration/__init__.py

```diff
@@ -171,15 +128,79 @@ def _tomlkit_inlinify(value: T) -> Union[T, Array, InlineTable]:
         return value
 
 
+_MEDIA_TYPE_REGEX = re.compile(
+    r"""
+    [A-Za-z0-9][A-Za-z0-9!#$&^_.+-]{,126}       # RFC 6838 type-name
+    /
+    [A-Za-z0-9][A-Za-z0-9!#$&^_.+-]{,126}       # RFC 6838 subtype-name
+    (?:
+        \s*                                     # optional whitespace (not technically allowed by the RFC
+                                                # but we see it in practice)
+        ;
+        \s*                                     # optional whitespace (not technically allowed by the RFC
+                                                # but we see it in practice)
+        [A-Za-z0-9][A-Za-z0-9!#$&^_.+-]{,126}   # RFC 2045 attribute
+        =
+        (?:[^\s]+|"[^"]+")                      # RFC 2045 value
+    )*
+    """,
+    re.VERBOSE,
+)
+
+
+def _looks_like_media_type(value: str) -> bool:
+    """
+    Return whether the string appears to represent a media type (MIME type).
+
+    For example:
+
+    >>> _looks_like_media_type("text/markdown")
+    True
+    >>> _looks_like_media_type("text/x-rst")
+    True
+    >>> _looks_like_media_type("text/plain")
+    True
+
+    This doesn't actually check whether the type is a real type that's registered
+    with IANA or in common use (use the :py:module:`mimetypes` module for that),
+    only whether it follows the syntax of a media type. So even unknown types
+    will return true.
```

Comment:
Probably safest doing this, people can and do invent their own weird and wonderful MIME types, as the system is extensible.

Reply:
Yeah, although custom types are supposed to follow specific formats (with `x.` or `x-` or similar).

I created #159 for adding a warning if the script detects a nonstandard content type, as a reminder to consider that.